libs/partners/huggingface/langchain_huggingface/embeddings/huggingface_endpoint.py

```diff
@@ -49,12 +50,9 @@ class Config:
     @root_validator(pre=False, skip_on_failure=True)
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that api key and python package exists in environment."""
-        values["huggingfacehub_api_token"] = get_from_dict_or_env(
-            values, "huggingfacehub_api_token", "HUGGINGFACEHUB_API_TOKEN", None
-        )
-
+        existing_key = next((key for key in ["HF_TOKEN", "HUGGINGFACEHUB_API_TOKEN"] if key in os.environ), "HUGGINGFACEHUB_API_TOKEN")
         huggingfacehub_api_token = get_from_dict_or_env(
```

Comment:
can we just make it `os.getenv("HF_TOKEN") or os.getenv("HUGGINGFACE_HUB_API_TOKEN") or None`? Does it need to be set in this, or is there a way for people to run an endpoint without auth?